### ISSUE TITLE
TODO: Sima(nemszerencse) kártyajátékok implementálása

### ISSUE THREAD

**Issue by Zentrom** [Zentrom/kartyajatek#3]
**Ide jöhet esetleg Admirális,Passziánsz,stb.**
A "KartyaJatek.java"-ba simán bekell irni a menübe, s külön classbe utána implementálni.
Valószinű lesz majd egy Cards/Kartyak(Deck/Pakli).java, amit meg lehet majd hivni attól függően hogy magyar vagy francia kártya-e.

**Comment by Zentrom** [comment]
"Treff-nem" játék akkor már megvan és megy is jól ahogy nézem.(max még megoldható benne hogy tényleg csak igen/nem input lehessen,mert most igy akkor jár legjobban a játékos ha spameli az "igen"-t)

**Comment by lipcseiadam** [comment]
igen, lényegében ez egy teszt arra hogy jól működik e a pakli és hasonló dolgok, viszont a bufferreader lezárására figyeljünk mert szenvedtem vele h miért hal le user input előtt :D

**Comment by Zentrom** [comment]
Ez az issue is lezárható lassan ha nem akartok több Casualgame-et implementálni. Admirális/Passziánsz nincs,ugyhogy ha valaki akarja kiszedheti őket a menüből...lehet kifogom én.
